src/sdk/pynni/nni/metis_tuner/requirments.txt

```diff
@@ -0,0 +1 @@
+sklearn
```

Comment:
Why you use a .txt file instead of a README.md?

Reply:
I think the requirement.txt list all the package we need to install.....

Reply:
It is all acceptable